src/main/java/com/springbootWithJPA/springBootRestExample/Employee.java

```diff
@@ -1,56 +1,17 @@
 package com.springbootWithJPA.springBootRestExample;
 
+import lombok.Data;
+
 import javax.persistence.*;
 
 @Entity
 @Table(name = "employees")
+@Data
 public class Employee {
```

Comment:
Since you have removed getters and setters.. please use @Getter and @setter annotation from lombok

Reply:
i have used @Data instead